torch/profiler/profiler.py

```diff
@@ -32,10 +32,33 @@
     "tensorboard_trace_handler",
     "profile",
     "ExecutionTraceObserver",
+    "NumpyEncoder",
```

Comment:
You can remove this entry now that it is not public anymore.

Reply:
removed!